tools/backup/values.yaml

```diff
@@ -1,8 +1,11 @@
 image: gcr.io/neo4j-helm/backup
 imageTag: 4.1.3-1
 neo4jaddr: holder-neo4j.default.svc.cluster.local:6362
+# In case of azure the bucket is used as a prefix for the container where the backup is stored
```

Comment:
Added the azure specific documentation here since it would be the place that I would look first.